Sources/Logging/Logging.swift

```diff
@@ -316,10 +316,9 @@ extension Logging.MetadataValue: ExpressibleByStringInterpolation {
             self = .string(strings.map { $0.description }.reduce("", +))
         }
 
-    // TODO: seems not to exist on 5.0-DEVELOPMENT-SNAPSHOT-2019-01-16-a
-    //        public init<T>(stringInterpolationSegment expr: T) {
-    //            self = .string(String(stringInterpolationSegment: expr))
-    //        }
+        public init<T>(stringInterpolationSegment expr: T) {
+            self = .string(String(stringInterpolationSegment: expr))
+        }    
```

Comment:
Whoops sorry about that! I can't read it seems, missed the `!` in `#if !swift(>=5.0)`